examples/helm/kanister/kanister-mysql/.helmignore

```diff
@@ -1 +1,2 @@
 .git
+OWNERS
```

Comment:
nit: add new line

Reply:
@SupriyaKasten I didn't get you. What is the change you are suggesting?

Reply:
I believe she wanted you to add a new line to the end of the file.

Reply:
got it, thanks!